runtime/doc/vim_diff.txt

```diff
@@ -354,7 +355,8 @@ Motion:
   The |jumplist| avoids useless/phantom jumps.
 
 Normal commands:
-  |Q| is the same as |gQ|
+  |Q| replays the last macro instead of switching to Ex mode.
```

Comment:
```suggestion
  |Q| replays the last recorded macro instead of switching to Ex mode.
```

Can you also add `RecordingEnter` and `RecordingLeave` to the `Events:` list under `FEATURES`?

Reply:
done in 9e24174423842ab39851a720b93963d1ce1c95b2